acquisitions/src/main/resources/thunderjet/consortia/features/open-order-with-locations-from-different-tenants.feature

```diff
@@ -0,0 +1,86 @@
+@parallel=false
+# for https://folio-org.atlassian.net/browse/MODORDSTOR-402
+Feature: Open ongoing order
+
+  Background:
+    * url baseUrl
+    * call login consortiaAdmin
+    * configure headers = { 'Content-Type': 'application/json', 'Authtoken-Refresh-Cache': 'true', 'x-okapi-token': '#(okapitoken)', 'x-okapi-tenant': '#(centralTenant)', 'Accept': 'application/json' }
+
+    * callonce variables
+
+    * def fundId = callonce uuid1
+    * def budgetId = callonce uuid2
+
+    * def orderId = callonce uuid3
+    * def orderLineIdOne = callonce uuid4
+
+  Scenario: Create fund and budget
+    * call createFund { 'id': '#(fundId)' }
+    * call createBudget { 'id': '#(budgetId)', 'allocated': 100, 'fundId': '#(fundId)', 'status': 'Active' }
+
+  Scenario: check budget after create
```

Comment:
Create fund and budget, check budget after create, Create orders, Create order lines - these calls should be wrapped in the same scenario, see guidlines here - https://folio-org.atlassian.net/wiki/spaces/FOLIJET/pages/114098928/Karate+Coding+Standards+Best+Practices

Reply:
In my case these parts are not scenarios at all but helper blocks to prepare data. Do we have some examples for this:
`- The feature file should look like it can executed in an existing tenant or a new tenant`